Added jupyter notebook references english

rakshith6404 opened the pull request:
PR-Hactoberfest2022
I have added Jupyter-english reference regarding #8121 
Looking forward to merge it :smile:

eshellman commented on more/free-programming-interactive-tutorials-en.md:
```diff
@@ -222,6 +223,12 @@
 * [Try jQuery](http://try.jquery.com)
 
 
+### Jupyter
```
these two resources should be in books, as a new category under python

eshellman replied:
move them to the books list

rakshith6404 commented:
I have made changes for the following please look into it.
I am looking forward to merge this file

rakshith6404 commented:
Hello Have added/Updated the requested changes yesterday 
i have added jupyter under python as a category , added related youtube tutorial(playlist) , added jupyter cheatsheet and also added jupyter youtube playlist in hindi .
https://github.com/rakshith6404/free-programming-books/blob/added_jupyter_notebook_references_english/more/free-programming-interactive-tutorials-en.md
https://github.com/rakshith6404/free-programming-books/blob/added_jupyter_notebook_references_english/courses/free-courses-hi.md
Looking forward for a review !!!! :smile:

eshellman commented:
Thanks, and welcome to free-programming-books!
